### PR TITLE
[SYCLomatic #732] Test ID and UUID migrations

### PR DESCRIPTION
**Pull request by 0x12CC**
Signed-off-by: Michael Aziz <michael.aziz@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: features/feature_case/device_info/device_info.cu [hunk 1]
```diff
@@ -7,14 +7,39 @@
...
+              << static_cast<int>(static_cast<unsigned char>(uuid.bytes[i]))
+              << " ";
+  }
+  std::cout << std::endl;
+}
+
+int main() {
+  test0();
+  test1();
+}
```
Comment (LU-JOHN) [inline features/feature_case/device_info/device_info.cu:45]: Does this need to return a value from main to report the status?

Comment (0x12CC) [reply features/feature_case/device_info/device_info.cu:45]: The test returns `0` (implicitly). We don't have expected values for the device query results so the test will always pass if it migrates, compiles, and runs without issues.

**Review by LU-JOHN — COMMENTED** [review]
LGTM

**Review by LU-JOHN — APPROVED** [review]

**Pull request merged**